build/bin/build-client-binary

```diff
@@ -35,7 +35,7 @@ gobuild_component() {
         export CGO_ENABLED=0
         export PMM_RELEASE_VERSION=${pmm_version}
         export PMM_RELEASE_TIMESTAMP=$(date '+%s')
-        export PMM_RELEASE_FULLCOMMIT=$(git rev-parse HEAD)
+        export PMM_RELEASE_FULLCOMMIT=$(cd sources/${component}/src/github.com/percona/${component} && git rev-parse HEAD)
```

Comment:
Why not `extract_path`?

Reply:
because for extract_path we need more changes and export parameter exactly before build command.
this construction works better

Reply:
@AlekSi done.